aten/src/ATen/native/cuda/PointwiseOpsKernel.cu

```diff
@@ -0,0 +1,22 @@
+#include <ATen/Context.h>
+#include <ATen/Dispatch.h>
+#include <ATen/native/cuda/Loops.cuh>
+#include <ATen/native/DispatchStub.h>
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/PointwiseOps.h>
+#include <THC/THCNumerics.cuh>
+
+namespace at { namespace native {
+
+void addcdiv_cuda_kernel(TensorIterator& iter, Scalar value) {
+  AT_DISPATCH_ALL_TYPES_AND(kHalf, iter.dtype(), "addcdiv_cuda", [&]() {
+    auto alpha = value.to<scalar_t>();
+    gpu_kernel(iter, [alpha]GPU_LAMBDA(scalar_t a, scalar_t b, scalar_t c) -> scalar_t {
+      return THCNumerics<scalar_t>::add(a, THCNumerics<scalar_t>::mul(alpha,THCNumerics<scalar_t>::div(b, c)));
```

Comment:
Do we still need to use THCNumerics? In https://github.com/pytorch/pytorch/blob/7f130c8494d235054917a7abfcbe1058a65d628a/aten/src/ATen/native/cuda/BinaryOpsKernel.cu#L19
We don't use it anymore.

Reply:
Fixed